test/harness/wasm2c/floating_point.txt

```diff
@@ -10,14 +9,10 @@
 (assert_return (invoke "f64" (f64.const 0.0)) (f64.const 123.4))
 (assert_return (invoke "f64" (f64.const 123.4)) (f64.const 0.0))
 (;; STDERR ;;;
-Error running "out/test/harness/wasm2c/floating_point/floating_point" (1):
-0/4 tests passed.
-
-out/test/harness/wasm2c/floating_point/floating_point-main.c:384: assertion failed: in w2c_floating__point__0__wasm_f32(&floating__point__0__wasm_instance, 0.f): expected 123.400002, got 0.
-out/test/harness/wasm2c/floating_point/floating_point-main.c:387: assertion failed: in w2c_floating__point__0__wasm_f32(&floating__point__0__wasm_instance, 123.400002f): expected 0, got 123.400002.
-out/test/harness/wasm2c/floating_point/floating_point-main.c:390: assertion failed: in w2c_floating__point__0__wasm_f64(&floating__point__0__wasm_instance, 0.0000000000000000): expected 123.40000000000001, got 0.
-out/test/harness/wasm2c/floating_point/floating_point-main.c:393: assertion failed: in w2c_floating__point__0__wasm_f64(&floating__point__0__wasm_instance, 123.40000000000001): expected 0, got 123.40000000000001.
-
+floating_point.txt:7: assertion failed: in w2c_floating__point__0__wasm_f32(&floating__point__0__wasm_instance, 0.f): expected 123.400002, got 0.
```

Comment:
I'm sorry for the late comment on this, but I really rely on these line numbers when debugging/developing, to find the failing assertion and the comment that leads to the line number in the original WAST file. Is there any way to keep these line numbers in our output in a cross-platform way?

Reply:
(The filename can definitely go; it's just the line number (e.g. the ":384") that I find valuable.)

Reply:
oh! we mean if you use "find in file" you can search the new files for e.g. `#line 7 "floating_point.txt"` to find the relevant C line where the assert was... (so it does require a change in workflow, but hopefully not too bad?)

there's no portable way to set the name without setting the line, unfortunately, so we grabbed the most convenient/appropriate line source (the wast line).